src/main/webapp/app/exercises/file-upload/assess/file-upload-assessment-dashboard.component.ts

```diff
@@ -23,6 +23,7 @@ export class FileUploadAssessmentDashboardComponent implements OnInit {
     busy = false;
     predicate = 'id';
     reverse = false;
+    getLatestSubmissionResult = getLatestSubmissionResult;
```

Comment:
wuuut :D

Reply:
This is used as a workaround to make the function available within the HTML. This will be gone once the refactoring is done next week!